js/logic.js

```diff
@@ -0,0 +1,91 @@
+const popupWrapper = document.querySelector('.project-popup-wrapper');
+const popupName = document.querySelector('.popup-name');
+const popupPosition = document.querySelector('.popup-position');
+const popupWorkName = document.querySelector('.popup-work-name');
+const popupImage = document.querySelector('.popup-image');
+const popupDescription = document.querySelector('.popup-description');
+const popupTech = document.querySelector('.popup-technologies-container');
+const projectOne = document.querySelector('#project-one');
+const projectTwo = document.querySelector('#project-two');
+const projectThree = document.querySelector('#project-three');
+const projectFour = document.querySelector('#project-four');
+const closePopup = document.querySelector('.close-project-popup');
+const blur = document.querySelector('#element');
+
+const popupOne = {
+  name: 'Tonic',
+  workName: 'CANOPY',
+  position: ['Back End Dev', '2015'],
+  imageUrl: './img/Snapshoot Portfolio.svg',
+  description:
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
+  technologies: ['html', 'css', 'javascript'],
```

Comment:
- [ ] Let's kindly make sure that all technologies are on the same line with the description on all your popups.